src/schema/index.js

```diff
@@ -32,12 +33,15 @@ type Oracle {
   amounts: [String!]!
   resultIdx: Int
   blockNum: Int!
+  startBlock: Int
```

Comment:
mark startBlock as `Int!` for consistency or filter in the future, if no value passed, use `blockNum`